Atomic/syscontainers.py

```diff
@@ -2522,9 +2540,11 @@ def _ensure_storage_for_image(self, repo, img):
         if manifest is None:
             raise ValueError("Image `%s` not present" % img)
         layers = SystemContainers.get_layers_from_manifest(json.loads(manifest))
+        SystemContainers._check_for_valid_layers(repo, layers)
```

Comment:
This check can also be dropped, if the perf impact for checking is large, but it is good to have :).